### PR TITLE
[backport][9.0] backport procurement analytic

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Added some extra features:

- Added tree view and search view in procurements
- Added a button in the analytic accounts to see the procurements associated.  This is useful for analyzing the procurements from a single analytic account

@carlosdauden @pedrobaeza @cubells if you agree i'll propose to v10 also

@eficent

### CODE REVIEW

File: procurement_analytic/models/account_analytic_account.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from openerp import api, fields, models
```
Comment (carlosdauden) [inline procurement_analytic/models/account_analytic_account.py]: api not used

**Review by carlosdauden — CHANGES_REQUESTED** [review]

File: procurement_analytic/models/account_analytic_account.py (hunk 1 above)
Comment (AaronHForgeFlow) [reply procurement_analytic/models/account_analytic_account.py]: Opps. I'll fix this

**Review by carlosdauden — APPROVED** [review]

**Comment by carlosdauden** [comment]
I delighted that you propose PR to v10.
Thanks.

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
@aheficent could you rebase the PR?

**Comment by AaronHForgeFlow** [comment]
You got it! :)

**Pull request merged**